src/PIL/ImageDraw.py

```diff
@@ -186,13 +186,13 @@ def polygon(self, xy, fill=None, outline=None):
         if ink is not None and ink != fill:
             self.draw.draw_polygon(xy, ink, 0)
 
-    def rectangle(self, xy, fill=None, outline=None):
+    def rectangle(self, xy, fill=None, outline=None, width=0):
```

Comment:
Since width is changed from 0 to 1 in the C layer, why not set the default to 1 here?

Reply:
👍